frontend/javascripts/libs/keyboard.ts

```diff
@@ -257,6 +257,7 @@
               this._paused = false;
               this.setContext("global");
               this.watch(targetWindow, targetElement, platform, userAgent);
+              this.setContext("default");
```

Comment:
We need a default context as otherwise the global context is default which will always be active